geode-core/build.gradle

```diff
@@ -320,6 +320,10 @@ dependencies {
     implementation(project(':geode-serialization'))
     implementation(project(':geode-tcp-server'))
 
+    runtimeOnly(project(':geode-deployment:geode-deployment-legacy')) {
+        ext.optional = true
+    }
+
```

Comment:
This declaration already exists on line 236, as a non-optional. With both in place, the result is a non-optional runtime dependency. Which is it?

If we chose optional, we'll also need to modify `test/resources/expected-pom.xml`

Reply:
Removed the duplicate.